tests/test_envs.py

```diff
@@ -40,6 +41,32 @@ def test_all_env_passive_env_checker(spec):
     env.close()
 
 
+@pytest.mark.parametrize(
+    "gym_id, mo_gym_id",
+    [
+        ("MountainCar-v0", "mo-mountaincar-v0"),
+        ("MountainCarContinuous-v0", "mo-mountaincarcontinuous-v0"),
+        ("LunarLander-v2", "mo-lunar-lander-v2"),
+        ("Reacher-v4", "mo-reacher-v4"),
+        ("Hopper-v4", "mo-hopper-v4"),
+        ("HalfCheetah-v4", "mo-halfcheetah-v4"),
+        ("Walker2d-v4", "mo-walker2d-v4"),
+        ("Ant-v4", "mo-ant-v4"),
+        ("Swimmer-v4", "mo-swimmer-v4"),
+        ("Humanoid-v4", "mo-humanoid-v4"),
+    ],
+)
+def test_gymnasium_equivalence(gym_id, mo_gym_id, num_steps=100, seed=123):
+    env = gym.make(gym_id)
+    mo_env = mo_gym.LinearReward(mo_gym.make(mo_gym_id))
```

Comment:
I have to check but I don't think all our environments just "split" the rewards into components